### PR TITLE
Test vars

### PR DESCRIPTION
**Pull request by InbalZilberman**
### What does this PR do?
Variable help has been updated
A test of list has changed to be more generic
A test of variable has changed to be more robust

### What ticket does this PR close?
Resolves #[relevant GitHub issues, eg 76]

#### Change log
- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ V] This PR includes new unit and integration tests to go with the code changes, or

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [V ] This PR does not require updating any documentation

### CODE REVIEW

**Comment by InbalZilberman** [comment]
1. I dont understand why i am behind master
2. Should I change changelog?

File: conjur/cli.py [hunk 1]
```diff
@@ -250,6 +274,7 @@ def launch():
         """
         Cli().run()
 
+
 if __name__ == '__main__':
     # Not coverage-tested since the integration tests do this
-    Cli.launch() # pragma: no cover
+    Cli.launch()  # pragma: no cover
```
Comment (sgnn7) [inline conjur/cli.py:280]: Unnecessary change

File: test/test_cli_integration.py [hunk 2]
```diff
@@ -31,6 +31,8 @@ class CliIntegrationTest(unittest.TestCase): # pragma: no cover
     HTTPS_CA_BUNDLE_ENV_VAR = { 'CONJUR_CA_BUNDLE': 'ca-bundle' }
 
     DEFINED_VARIABLE_ID = 'one/password'
+    DEFINED_VARIABLE2 = 'two/username'
+    DEFINED_VARIABLE3 = 'three/secret'
```
Comment (sgnn7) [inline test/test_cli_integration.py:35]: We should indicate that they are variable IDs like the initial L33 one:
```suggestion
    DEFINED_VARIABLE_ID_2 = 'two/username'
    DEFINED_VARIABLE_ID_3 = 'three/secret'
```

File: test/test_cli_integration.py [hunk 3]
```diff
@@ -166,7 +179,14 @@ def test_https_cli_can_set_and_get_a_defined_variable(self):
...
         })
 
+        self.set_variable(CliIntegrationTest.DEFINED_VARIABLE2, "r")  # "!@#$%asdfgh&*()g,lpokmnjiub")
+        self.set_variable(CliIntegrationTest.DEFINED_VARIABLE3, "1")
+        output = self.get_variable([CliIntegrationTest.DEFINED_VARIABLE2, CliIntegrationTest.DEFINED_VARIABLE3])
         self.assert_set_and_get(CliIntegrationTest.DEFINED_VARIABLE_ID)
+        self.assert_json_equals(output,
+                                "{ \"" + CliIntegrationTest.DEFINED_VARIABLE2 + "\": \"r\", \"" + CliIntegrationTest.DEFINED_VARIABLE3 + "\": \"1\" }")
+
+
```
Comment (sgnn7) [inline test/test_cli_integration.py:189]: Too many empty lines

**Comment by sigalsax** [comment]
closing this is stale
